client/quantum_serverless/core/tracing.py

```diff
@@ -114,3 +114,17 @@ def _trace_env_vars(env_vars: dict, location: Optional[str] = None):
         env_vars[OT_TRACEPARENT_ID_KEY] = traceparent
         os.environ[OT_TRACEPARENT_ID_KEY] = traceparent
     return env_vars
+
+
+def setup_tracing() -> None:
+    """Setup Tracing for Ray cluster
+
+    Passed as an argument at Ray start
+    """
+    resource = Resource(attributes={SERVICE_NAME: "Quantum-Serverless: Ray"})
+    provider = TracerProvider(resource=resource)
+    otel_exporter = BatchSpanProcessor(
+        OTLPSpanExporter(endpoint="http://otel-collector:4317/", insecure=True)
+    )
+    provider.add_span_processor(otel_exporter)
+    trace._set_tracer_provider(provider, log=False)
```

Comment:
think the lint warning can be disabled via something like  `# pylint: disable=W0212` , but should we? not sure what the python best practices are in this space...

Reply:
looks like you fixed it while I was typing that :laughing: